### PR TITLE
Cleanup CFN template and parameters

### PR DESCRIPTION
**Pull request by drAlberT**

### CODE REVIEW

**Review by AlessandroMinoccheri — APPROVED** [review]
LGTM 👍

File: sys/cloudformation/stack.yaml [hunk 1]
```diff
@@ -165,370 +230,343 @@ Resources:
...
-      LogGroupName:
-        Fn::Sub:
-          - "${ServiceName}-logs"
+      LogGroupName: !Sub '${ServiceName}-logs'
+      RetentionInDays: 30
 
   # ELB
   elb:
     Type: AWS::ElasticLoadBalancingV2::LoadBalancer
-    DependsOn:
```
Comment (fabiocicerchia) [inline sys/cloudformation/stack.yaml:203]: is this useless? just to know more about it ;)

File: sys/cloudformation/stack.yaml [hunk 2]
```diff
@@ -165,370 +230,343 @@ Resources:
...
+            - !Ref sgecs
       Tags:
         - Key: env
-          Value: badge-poser
+          Value: !Ref ServiceName
 
   # ECS TASK DEFINITION
   ecstask:
-    Type: 'AWS::ECS::TaskDefinition'
-    DependsOn:
```
Comment (fabiocicerchia) [inline sys/cloudformation/stack.yaml:296]: is this useless? just to know more about it ;)

File: sys/cloudformation/stack.yaml (hunk 1 above)
Comment (drAlberT) [reply sys/cloudformation/stack.yaml:203]: Yes, as the dependence is already stated by the `!Ref` in "SecurityGroups" property

File: sys/cloudformation/stack.yaml (hunk 2 above)
Comment (drAlberT) [reply sys/cloudformation/stack.yaml:296]: Yep, the dependence is already enforced by this:
```

          LogConfiguration:
            LogDriver: awslogs	            LogDriver: awslogs
            Options:	            Options:
              awslogs-group: !Ref 'cloudwatchloggroup'	              awslogs-group: !Ref cloudwatchloggroup
```

**Review by fabiocicerchia — APPROVED** [review]

**Pull request merged**